pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -19,17 +19,37 @@
 
 
 @pytest.fixture(scope="session")
-def nrao_uv():
+def nrao_uv_main():
+    # This file is known to be made with CASA's importuvfits task, but apparently the
+    # history table is missing, so we can't infer that from the history. This means that
+    # the uvws are not flipped/data is not conjugated as they should be. Fix that.
```

Comment:
Maybe we should add a check to read_uvfits that is similar to the check at the end of read_ms to see if a flip needs to happen.

Reply:
So the check is in `ms.py` (the "this is a thing" comment marks where it is currently) -- I can certainly add a similar check to UVFITS, although if we're talking about expanding it to multiple filetypes, perhaps this is something that should be added to `UVData`? Either as a standalone method or as a piece of code in `check` that can be activated with a keyword?

Reply:
Sure, that's probably the right way to implement it. I guess that this can be flipped easily because the images just work out.

Reply:
Migraded, and the commented-out code has been deleted.